src/condition.cpp

```diff
@@ -1041,6 +1092,10 @@ conditional_t<T>::conditional_t( const JsonObject &jo )
         set_compare_var( jo, "u_compare_var" );
     } else if( jo.has_string( "npc_compare_var" ) ) {
         set_compare_var( jo, "npc_compare_var", is_npc );
+    } else if( jo.has_string( "u_compare_var_time" ) ) {
+        set_compare_var_time( jo, "u_compare_var_time" );
+    } else if( jo.has_string( "npc_compare_var_time" ) ) {
+        set_compare_var_time( jo, "npc_compare_var_time", is_npc );
```

Comment:
I think it would be clearer what this is doing (and how it's different from the other comparison) if it was called `compare_time_since_var`.

Reply:
Excellent idea, thanks!